modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -40,11 +40,37 @@ definition {
 	@description = "This is a test for COMMERCE-6030. AddAnAttachment"
 	@ignore = "true"
 	@priority = "3"
-	test AddAnAttachment {
+	test EditAnAttachment {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6030 AddAnAttachment
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
```

Comment:
The Master catalog is already present because on line 20 into the setUp we have a reindex